app/templates/logos.hbs

```diff
@@ -3,7 +3,7 @@
   <section aria-labelledby="branding" class="branding-section">
     <h1 id="branding">Branding</h1>
     <p>The Ember brand and associated assets are part of a meticulously curated experience. We take great pride in what our team and community have helped bring to life, and ask that you follow our guidelines to ensure consistency across all Ember properties.</p>
-    <p><a href="/pdfs/Ember-Brand-Guidelines.pdf">Download brand guidelines (PDF)</a></p>
+    <p><a href="https://emberjs.com/pdfs/Ember-Brand-Guidelines.pdf">Download brand guidelines (PDF)</a></p>
```

Comment:
isn't https://github.com/ember-learn/ember-website/pull/1052 the fix?

if we add a domain here, then we can't see preview versions of this pdf.
I think we want to be able to view subdomain versions of the pdf, like in the other PR?
![image](https://github.com/ember-learn/ember-website/assets/199018/431e20bc-e8a0-4326-a41b-361696e9f660)